pkg/controller/bastion/bastion_suite_test.go

```diff
@@ -0,0 +1,320 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bastion
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"runtime"
+	"testing"
+
+	mockgcpclient "github.com/gardener/gardener-extension-provider-gcp/pkg/internal/mock/client"
+	"github.com/gardener/gardener/extensions/pkg/controller"
+	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	extensionsv1alpha1 "github.com/gardener/gardener/pkg/apis/extensions/v1alpha1"
+	"github.com/gardener/gardener/pkg/extensions"
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"google.golang.org/api/compute/v1"
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	runtime2 "k8s.io/apimachinery/pkg/runtime"
+)
+
+func TestBastion(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Bastion Suite")
+}
+
+var _ = Describe("Bastion", func() {
+	var (
+		cluster *extensions.Cluster
+		bastion *extensionsv1alpha1.Bastion
+
+		opt  Options
+		ctrl *gomock.Controller
+	)
+	BeforeEach(func() {
+		cluster = createGCPTestCluster()
+		bastion = createTestBastion()
+		ctrl = gomock.NewController(GinkgoT())
+	})
+	AfterEach(func() {
+		ctrl.Finish()
+	})
+
+	Describe("getWorkersCIDR", func() {
+		It("getWorkersCIDR", func() {
+			cidr, err := getWorkersCIDR(createGCPTestCluster())
+			Expect(err).To(Not(HaveOccurred()))
+			Expect(cidr).To(Equal("10.250.0.0/16"))
+		})
+	})
+
+	Describe("Determine options", func() {
+		It("should return options", func() {
+			options, err := DetermineOptions(bastion, cluster, "projectID")
+			if err != nil {
+				fmt.Println(err)
+			}
```

Comment:
```suggestion
			Expect(err).To(Not(HaveOccurred()))
```

Reply:
committed in https://github.com/gardener/gardener-extension-provider-gcp/pull/288/commits/6ee5df4bf19f075d65c030c90aca558b5335ed07